synapse/lib/cell.py

```diff
@@ -680,6 +733,76 @@ async def getNexusChanges(self, offs):
         async for item in self.nexsroot.iter(offs):
             yield item
 
+    def _reqBackDirn(self, name):
+        self._reqBackConf()
+
+        path = s_common.genpath(self.backdirn, name)
+        if not path.startswith(self.backdirn):
+            mesg = 'Directory traversal detected'
+            raise s_exc.BadArg(mesg=mesg)
+
+        return path
+
+    async def runBackup(self, name=None, wait=True):
+
+        if name is None:
+            name = time.strftime('%Y%m%d%H%M%S', datetime.datetime.now().timetuple())
+
+        path = self._reqBackDirn(name)
+        if os.path.isdir(path):
+            mesg = 'Backup with name already exists'
+            raise s_exc.BadArg(mesg=mesg)
+
+        task = self.schedCoro(self._execBackTask(path))
+
+        if wait:
```

Comment:
Probably not the scope of this PR, but it would be good to have an indicator if the backups completed successfully